lib/usd/ui/layerEditor/layerEditorWidget.cpp

```diff
@@ -221,16 +220,21 @@ QLayout* LayerEditorWidget::setupLayout_toolbar()
     toolbar->setContentsMargins(margin, 0, 0, 0);
     auto buttonAlignment = Qt::AlignLeft | Qt::AlignRight;
 
-    auto addButton = [toolbar, buttonAlignment](const char* iconName, const QString& tooltip) {
-        auto icon = utils->createIcon(iconName);
-        auto button = new GeneratedIconButton(nullptr, icon);
-        button->setToolTip(tooltip);
-        toolbar->addWidget(button, 0, buttonAlignment);
-        return button;
+    auto addHIGButton = [buttonSize, toolbar, buttonAlignment](const QString& iconName, const QString& tooltip) {
+        auto saveButtonYOffset = DPIScale(4);
+        auto saveButtonSize = QSize(buttonSize + DPIScale(12), buttonSize + saveButtonYOffset);
+        auto higBtn = new QPushButton();
+        higBtn->move(0, saveButtonYOffset);
+        setupButtonWithHIGBitmaps(higBtn, iconName);
```

Comment:
Use the same HIG setup function as the "save" button uses - it was already a HIG button.